ros/lib/models.py

```diff
@@ -55,6 +55,7 @@ class System(db.Model):
     rule_hit_details = db.Column(JSONB)
     state = db.Column(db.String(25))
     number_of_recommendations = db.Column(db.Integer)
+    stale_timestamp = db.Column(db.String(50))
```

Comment:
@r14chandra  Since it is a Date type field, we should use a different type to store the value.
```suggestion
    stale_timestamp = db.Column(db.DateTime)
```
@kgaikwad do we need to provide any default value in case of `stale_timestamp`?

Reply:
@apuntamb - No default value is required here.